### PR TITLE
implement session_timeout at the module level

### PR DESCRIPTION
**Pull request by goneri**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1073

Don't use a global value to set the timeout. This works poorly with the cached session.

### CODE REVIEW

File: vmware_rest_code_generator/templates/header.j2 [hunk 1]
```diff
@@ -73,10 +74,11 @@ def prepare_argument_spec():
             fallback=(env_fallback, ['VMWARE_REST_LOG_FILE']),
         ),
         "session_timeout": dict(
-            type='float',
+            type="float",
             default=300,
```
Comment (abikouo) [inline vmware_rest_code_generator/templates/header.j2:78]:
```suggestion
```

Comment (abikouo) [reply vmware_rest_code_generator/templates/header.j2:78]: I think we should remove this default value so that we keep the current way of working

Comment (goneri) [reply vmware_rest_code_generator/templates/header.j2:78]: ok.

**Comment by goneri** [comment]
recheck

**Pull request merged**